### ISSUE TITLE
No signature verification

### ISSUE THREAD

**Issue by federicofusco** [nathan-149/CustomCrypocurrency#16]
I was looking through the `blockchain.py` file and noticed that the `isValidTransaction()` function which should verify the validity of the transaction doesn't actually check that signature so anybody can forge transactions on someones behalf using a fraudulent private key.

**Comment by batzuu** [comment]
Yep I also found this issue, also in `routes.py` at line `33` the `addTransaction()` function has two same arguments for `keyString` and `senderKey` which is `form.key,data` which is the stored public key of the user. Also I could not seem to find the where he used the `private.pem` after storing the private key.

**Comment by NTechStorm** [comment]
I'm pretty sure he fixed this error recently.
